src/PortingAssistantExtensionServer/Services/AnalysisService.cs

```diff
@@ -167,6 +168,10 @@ public bool HasSolutionAnalysisResult()
 
                 foreach (var projectAnalysisResult in solutionAnalysisResult.ProjectAnalysisResults)
                 {
+                    if (projectAnalysisResult.ProjectFilePath == null) {
```

Comment:
Should we be using string.IsNullOrEmpty here since the DocumentUri.FromFileSystemPath will throw on empty string as well?

Reply:
good point! Will update it to `IsNullOrEmpty`.